control.lua

```diff
@@ -82,7 +81,13 @@ script.on_event(defines.events.on_gui_opened, TeleporterGUI.onGuiOpened)
 
 script.on_event(defines.events.on_gui_closed, TeleporterGUI.onGuiClosed)
 
-script.on_event(defines.events.on_gui_click, TeleporterGUI.onGuiClick)
+script.on_event(defines.events.on_gui_click, function(event)
+  if event.element.name == "coe_teleporter_gui_close" or next( event.element.tags ) ~= nil then
```

Comment:
The event handler in each file should handle this logic. I believe it is already handled for Teleporter
Also the stats_gui script has not been added

Reply:
Additionaly I think its time to add simple event handler!

Reply:
So control.lua can have multiple `script.on_event( ... )` for the same event?

Reply:
Only one per event,  but each of our script files can have an onXXX handler that we call inside the script.on_event